lib/sensu/spawn.rb

```diff
@@ -159,8 +159,14 @@ def child_process(command, options={})
         end
         [output, child.exit_code]
       rescue ChildProcess::TimeoutError
-        child.stop rescue nil
-        ["Execution timed out", 2]
+        output = "Execution timed out"
+        begin
+          child.stop
+        rescue => error
+          pid = child.pid rescue "?"
+          output += " - Unable to TERM/KILL the process: ##{pid}, #{error}"
```

Comment:
does `##{}` have some special meaning or is that a typo?

Reply:
oh maybe you meant since its an actual number you want it to look like `process: #1234`

Reply:
@majormoses outputs #$PID